src/API.CRUD/Models/ApiError.cs

```diff
@@ -0,0 +1,69 @@
+using System;
+using System.Collections.Generic;
+
+namespace API.CRUD
+{
+    public class ApiError

```

Comment:
For discussion. I'd like to do more research on how to best represent API errors, both from a JSON standpoint as well as the code.

Reply:
🔧